modin/pandas/dataframe.py

```diff
@@ -255,32 +241,9 @@ def drop_duplicates(
 
     @property
     def dtypes(self):
-        """Get the dtypes for this DataFrame.
-
-        Returns:
-            The dtypes for this DataFrame.
-        """
         return self._query_compiler.dtypes
 
     def duplicated(self, subset=None, keep="first"):
-        """
```

Comment:
Why are docstrings for multiple methods deleted? Whereas, there are docstrings for those methods in pandas.

Reply:
These docstrings is added to methods using `_inherit_docstrings` decorator. We do not need to manually copy them, otherwise, when moving to a new pandas release, we will face the fact that the documentation for some methods / functions will be irrelevant.

Reply:
See comment to previous discussion.